packages/rich-text/src/component/index.native.js

```diff
@@ -363,6 +364,10 @@ export class RichText extends Component {
 			return;
 		}
 
+		if ( KEYCODES_MOBILE[ event.keyCode ] ) {
+			event.code = KEYCODES_MOBILE[ event.keyCode ];
+		}
```

Comment:
My understanding is that this mapping is a fairly permanent change based upon the architecture of Aztec. Is that accurate? 

Given that Aztec manages [setting of `keyCode`](https://github.com/WordPress/gutenberg/blob/8888efea0a1629598becdbe40b4252dd01c835c6/packages/react-native-aztec/src/AztecView.js#L105), does it make more sense to house this logic in Aztec? 

Semi-related, Aztec [depends upon `@wordpress/keycodes`](https://github.com/WordPress/gutenberg/blob/8888efea0a1629598becdbe40b4252dd01c835c6/packages/react-native-aztec/src/AztecView.js#L15).

Reply:
Good suggestion! I think it would make more sense in Aztec yes. I'll move it there and if in the future we need it somewhere else we could move it. Thanks!